db/migrate/20171206230107_convert_template_links_to_text.rb

```diff
@@ -0,0 +1,10 @@
+class ConvertTemplateLinksToText < ActiveRecord::Migration
+  def up
+    # BLOB/TEXT fields cannot have a default value - changing to set default in app/models/template.rb
+    change_column :templates, :links, :text, default: nil
```

Comment:
I don't see a case right now to change the type from VARCHAR to TEXT. As far as I'm aware both postgresl and mysql can have a variable length VARCHAR which should satisfy the requirements to store the links object as string.

Reply:
Unless we have a string greater than 65535 which probably we never have...

Reply:
Hmm interesting. If you go in and add 3 funder links and 2 sample plans you get
![screen shot 2017-12-07 at 8 34 50 am](https://user-images.githubusercontent.com/1204467/33726943-60bbe400-db2b-11e7-8200-6e0aff2a0026.png)

All of the varchars in our DBs here are limited to 255 (which was an older limit for MySQL pre 5.0). We're on 5.7 right now which, as you say allows 65k (depending on total row size), so we should be able to increase them on our end.

Its likely something carried over from the original DB we started with at the beginning of the project. I'll have to do some research to determine the best way for us to remove those limitations. In the interim, I'll back out the migration and move of the default to the code.

Reply:
@briri if running your migration solves the problem quickly that's fine but it would be nice to keep VARCHAR since we never going to have huge content for that field.